RxWebKit.xcodeproj/project.pbxproj

```diff
@@ -565,7 +565,7 @@
 				PRODUCT_NAME = RxWebKit;
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 5.0;
```

Comment:
No reason to do this - We support Swift 4.2 and Swift 5.0 in Xcode 10.2.
Let's put the lower version of the two (4.2).

Reply:
But I try to pod install RxWebKit on other projects will prompt RxWebKit only supports RxSwift 4.0.

Reply:
RxSwift 5 supports Swift 4.2, so this shouldn't be a problem.

Reply:
You can check out my issue: https://github.com/RxSwiftCommunity/RxWebKit/issues/28

Reply:
I saw the issue. But you updated from 4.0 to 5.0. 
You need to update from 4.0 to 4.2.

Reply:
This is not the case. At present, the project is already 5.0. At this time, pod install RxWebKit will appear in the issue.

Reply:
Sorry, I don't understand. I meant that in this PR you bumped from 4.0 to 5.0, there's no reason to do this in RxWebKit unless I'm misunderstanding.

Reply:
And I tried to use CocoaPods search RxWebKit, only showing version 0.3.7, can't find 0.3.8.

Reply:
Sorry, it may be the problem I expressed, but I have tried several Projects, they have been upgraded to RxSwift 5.0 before, but the same problem as #28 in pod install RxWebKit, or you try again?

Reply:
The latest version has not been trunked that's the problem. @mokumoku can you add the community email address so I can trunk the released version? Thanks

Reply:
So....How can I fix?

Reply:
@bobgodwinx So sorry to be late my response.
I can maybe add community email address but please wait until Monday because of my vacation 🙇

Reply:
I added `community@rxswift.org` to the owner!
Please release latest version 1.0.0 :pray:
I create a tag 1.0.0 but I can't push because my Xcode is old (10.1) :cry:

```
☁  RxWebKit [master] pod trunk info RxWebKit

RxWebKit
    - Versions:
      - 0.1.0 (2016-02-12 13:32:17 UTC)
      - 0.2.0 (2016-03-01 15:53:37 UTC)
      - 0.2.2 (2016-07-14 17:37:23 UTC)
      - 0.3 (2016-10-02 07:59:12 UTC)
      - 0.3.1 (2016-10-13 17:34:27 UTC)
      - 0.3.2 (2016-10-20 17:48:39 UTC)
      - 0.3.3 (2016-10-25 06:12:56 UTC)
      - 0.3.4 (2017-02-15 11:46:39 UTC)
      - 0.3.5 (2017-09-25 19:42:55 UTC)
      - 0.3.6 (2017-10-29 18:13:01 UTC)
      - 0.3.7 (2018-03-04 10:24:12 UTC)
    - Owners:
      - Daichi Ichihara <da1lawmoku2@gmail.com>
      - RxSwift Community <community@rxswift.org>
```

Thanks!

Reply:
omg...This is not a very good thing.😢

Reply:
I will take care of it asap. No worries

Reply:
ok~thanks~